spec/client_spec.rb

```diff
@@ -192,44 +183,44 @@
     context 'when skip_auth is not nil' do
       it 'does not authorize before proceeding' do
         response = double('response')
-        allow(RestClient::Request).to receive(:execute).and_return(response)
+        allow(Vra::Http).to receive(:execute).and_return(response)
         expect(client).to_not receive(:authorize!)
 
         client.http_head('/test', :skip_auth)
       end
     end
 
-    it 'calls RestClient::Request#execute' do
+    it 'calls Vra::Http#execute' do
```

Comment:
Pedantic: but `#` should be reserved for instance methods, and technically `execute` is a class method in `Vra::Http`. I'd update this to `calls Vra::Http.execute`